managed/models/settings_helpers.go

```diff
@@ -132,11 +139,22 @@ func SetPMMServerID(q reform.DBTX) error {
 
 // UpdateSettings updates only non-zero, non-empty values.
 func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, error) {
```

Comment:
I wonder why this type is used here for `q`. It seems that we can use `*reform.TX` here and eliminate type check.

Reply:
The `reform.DBTX` is later used when getting settings.
If we were to change it to `reform.TX` we would have to change `GetSettings()` to `reform.TX` as well which would mean we would need to retrieve settings in a transaction. This would affect all other places where `GetSettings()` is called.
I opted for asserting instead as it seems to make more sense in this particular case.